RISE-V2G-EVCC/src/main/java/com/v2gclarity/risev2g/evcc/transportLayer/TLSClient.java

```diff
@@ -220,6 +221,10 @@ public void stop() {
 		}
 	}
 	
+	private static void releaseInstance() {
+		uniqueTLSClientInstance = null;
+	}
+	
```

Comment:
Same here, see my comment for TCPClient.java

Reply:
no problem, reorganized this.